### PR TITLE
Hopf torus

### PR DESCRIPTION
**Pull request by stla**
An example: a Hopf torus with a dynamic texture (#2078).

### CODE REVIEW

**Comment by MatthewFlamm** [comment]
Windows testing is broken currently but is hopefully fixed in #2137.  Don't worry about those checks

**Comment by banesullivan** [comment]
Hi @stla, thank you for your contributions!! Please disregard the `Pull Request Labeler / triage (pull_request)` check

File: examples/02-plot/moving_cmap.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+"""
+
+from math import pi
+
+import numpy as np
+
+import pyvista as pv
+
+# A spherica curve
```
Comment (adeak) [inline examples/02-plot/moving_cmap.py:17]:
```suggestion
# A spherical curve

```

File: examples/02-plot/moving_cmap.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+x, y, z = F(u, v)
+grid = pv.StructuredGrid(x, y, z)
+mesh = grid.extract_geometry().clean(tolerance=1e-6)
+
+# Distances normalized to [0, 2*pi]
+dists = np.linalg.norm(mesh.points, axis=1)
+dists = 2 * pi * (dists - dists.min()) / (dists.max() - dists.min())
+
+# Make the movie
+pltr = pv.Plotter(window_size=[512,512])
```
Comment (adeak) [inline examples/02-plot/moving_cmap.py:64]:
```suggestion
pltr = pv.Plotter(window_size=[512, 512])

```

**Review by adeak — COMMENTED** [review]
This is awesome! Thanks for posting this.

Aside from the below trivial typo-grade nitpicks I only have a few general remarks, none of which are crucial:
1. I'd probably use `np.pi` if we already have NumPy imported.
2. I think (hope) our examples quite rigorously follow the `snake_case` function naming convention of the library, so perhaps that would make this example fit even more in.
3. This is 100% subjective, but I loved the "`nipy_spectral`" version from your discussion post a lot. The flag colormap version seems a bit trippy to me, I could maybe imagine people being uncomfortable looking at it. But the `nipy_spectral` is very calm and smooth.

Again, none of these are crucial. This is an excellent example as is.

File: examples/02-plot/moving_cmap.py (hunk 1 above)
Comment (stla) [reply examples/02-plot/moving_cmap.py:17]: Ok, I do the changes. Me too, I like the result with the `nipy_spectral` colormap.

**Review by adeak — APPROVED** [review]
Looks great, thanks again @stla.

**Pull request merged**